nexus_demos/launch/launch.py

```diff
@@ -88,12 +88,16 @@ def launch_setup(context, *args, **kwargs):
                         pick_and_place: [place_on_conveyor, pick_from_conveyor],
                     }"""
     rviz_config_filename = "nexus_panel.rviz"
+    max_jobs = "2"
+    max_workcell_jobs = "1"
     if (use_rmf_transporter.perform(context).lower() == "true"):
         remap_task_types = """{
                             pick_and_place_rmf: [place_on_conveyor, pick_from_conveyor],
                         }"""
         main_bt_filename = "main_rmf.xml"
         rviz_config_filename = "nexus_panel_rmf.rviz"
+        max_jobs = "10"
+        max_workcell_jobs = "10"
```

Comment:
I think we want to set these to 10 only for the RMF workcell. The pick & place workcells will still have 1 configured.

Reply:
Per discussion, it's 1 by default, https://github.com/osrf/nexus/pull/73/files#diff-5c27d59c3eaa555d9589cb6aa673aeefc6806e2efe6d666a3f0e560fc7cab95eR92